irods/access.py

```diff
@@ -1,5 +1,42 @@
+import collections
+
 class iRODSAccess(object):
 
+    codes = { key:value for key,value in dict(  # copied from iRODS server code:
+        null                 = 1000,            # server/core/include/irods/catalog_utilities.hpp
+        execute              = 1010,
+        read_annotation      = 1020,
+        read_system_metadata = 1030,
+        read_metadata        = 1040,
+        read_object          = 1050,
+        write_annotation     = 1060,
+        create_metadata      = 1070,
+        modify_metadata      = 1080,
+        delete_metadata      = 1090,
+        administer_object    = 1100,
+        create_object        = 1110,
+        modify_object        = 1120,
+        delete_object        = 1130,
+        create_token         = 1140,
+        delete_token         = 1150,
+        curate               = 1160,
+        own                  = 1200
+    ).items() if key in (
```

Comment:
I still think that an additional indentation could be helpful since this closing parenthesis is at the same level of indentation as the encompassing curly braces (see lines 5 and 36).

Reply:
Can always run the python-black formatter on it to see what is produced.

I'm not a fan of the additional indentation, but I understand what you're getting at.

As an alternative, the dicts and lists could be externalized into local variables to avoid indentation altogether.

Feel free to ignore my comments here. I'm just presenting ideas.

Reply:
Yeah, I'm fine with externalizing the dicts as well as just leaving it alone.

Reply:
I think the add'l indent, while not "regular", isn't too offensive.  The thing as a whole is simple and succinct, so I'm happy.Will squash to 1 commit.